### DISCUSSION TITLE
[MAC] Problems with thumbnail generator.

### DISCUSSION THREAD

**Discussion by parkbom21** [smplayer-dev/smplayer#677] [General]
When I generate a thumbnail, everything is fine... Except one thing. The video mediainfo doesn’t show...
![GOAWAY_preview](https://user-images.githubusercontent.com/128273148/226152742-0baf22ec-5625-497e-8e1c-03b910bddccc.jpg)

**Comment by RykRok** [comment]
I'm having the same problem, what I had to do was put my Linux in light mode as dark mode was forcing SMPlayer to use white fonts on the white background, by going back to light mode, the info was now present in a black font.
It would be nice if we could change the color of fonts independently in SMPlayer.

**Comment by smplayer-dev** [comment]
Could you check if the problem is fixed in this build? (the artifact at the bottom)

https://github.com/smplayer-dev/smplayer/actions/runs/9465081915

**Reply by parkbom21** [reply]
Installed the build and it seems to have been fixed, thank you!

**Comment by RykRok** [comment]
Thank you for the info, unfortunately at the moment I'm having a heck of a
time with my Endeavour OS on my removable OS drive, the linux boot option
keeps disappearing from my system boot menu, until I can fix this problem I
can't really test your smplayer solution,
Again thanks so much for getting back to me, I'll keep you posted
